src/tests/java/htsjdk/samtools/fastq/FastqRecordTest.java

```diff
@@ -0,0 +1,136 @@
+package htsjdk.samtools.fastq;
```

Comment:
Crap, totally my fault for not remembering that this was a test file, just a habit.